vg/len.go

```diff
@@ -29,3 +35,41 @@ func (l Length) Dots(dpi float64) float64 {
 func (l Length) Points() float64 {
 	return float64(l)
 }
+
+func (l *Length) String() string {
```

Comment:
Do we need Length to be a fmt.Stringer? I know it was necessary for flag.Value implementation, but we're not doing that now.